src/client/pages/login.jsx

```diff
@@ -32,6 +34,25 @@ const Login = () => {
                 padding-top: 40px;
                 padding-bottom: 40px;
               }
+
+              .bg {
```

Comment:
This part is almost the same as the style of SignUpBox so if we can extract this part, it is pretty nice!
But for the design it looks good to me.